hal/riscv64/arch/cpu.h

```diff
@@ -18,20 +18,20 @@
 
 #include "hal/types.h"
 
-#define SIZE_PAGE 0x1000
+#define SIZE_PAGE 0x1000U
```

Comment:
`U` suffix usage has to be verified
```suggestion
#define SIZE_PAGE 0x1000UL
```

Reply:
Corrected.